src/AzureAuth/CommandMain.cs

```diff
@@ -262,10 +267,52 @@ private int GetToken()
             try
             {
                 ITokenFetcher tokenFetcher = this.TokenFetcher();
-                TokenResult tokenResult = (this.tokenFetcherOptions.Scopes == null
-                    ? tokenFetcher.GetAccessTokenAsync(this.CombinedAuthMode)
-                    : tokenFetcher.GetAccessTokenAsync(this.tokenFetcherOptions.Scopes, this.CombinedAuthMode))
-                    .Result;
+                TokenResult tokenResult = null;
+
+                // When running multiple AzureAuth processes with the same resource, client, and tenant IDs,
+

```

Comment:
Looks like there's an accidental extra newline here. I think you can just delete this line.